apps/trading/components/liquidity-supplied/liquidity-supplied.tsx

```diff
@@ -107,13 +108,26 @@ export const MarketLiquiditySupplied = ({
     },
   ];
 
+  const showMessage =
+    percentage.gte(100) &&
+    market?.marketTradingMode ===
+      MarketTradingMode.TRADING_MODE_MONITORING_AUCTION &&
+    market.trigger === AuctionTrigger.AUCTION_TRIGGER_LIQUIDITY;
+
   const description = (
     <section>
       {compiledGrid && <DataGrid grid={compiledGrid} />}
       <br />
       <Link href={`/#/liquidity/${marketId}`} data-testid="view-liquidity-link">
         {t('View liquidity provision table')}
       </Link>
+      {showMessage && (
+        <p className="mt-4">
+          {t(
+            'If there is sufficient liquidity, the market is in an auction because there are no priced limit orders, which are required to deploy liquidity provision pegged orders. This means the order book is empty on one or both sides.'
```

Comment:
Will a sentence be displayed that also says the market is in an auction because there isn't enough supplied liquidity? Without that, it reads alike something's missing.

Alternatively, I think you could cut the first part of the sentence so it reads: `The market is in an auction because there are no priced limit orders, which are required to deploy liquidity commitment pegged orders. This means the order book is empty on one or both sides.`

Reply:
Nice! I like that. But I should check with @barnabee too.

Reply:
The sentence will be displayed only when supplied > target.